docs/Examples/MultipartUpload.md

```diff
@@ -16,7 +16,7 @@ Here, our additional parameter is `description`, which is a `String`.
 
 ## Parameters in body
 
-When we want to add parameters to the body of our request, we need to append them as a `MultipartFormData` in the `task` property:
+When we want to perform multipart upload request with additional parameters in request body, we have to create a `MultiPartFormatData` for each of our parts and them return a `.uploadMultipart(_:)` in the `task` property:
```

Comment:
nit: 
perform *a* multipart
parameters in *the* request body
them => then

Reply:
Done, good catches! 👍